adapters/parent/pom.xml

```diff
@@ -248,5 +258,131 @@
+            <executions>
+              <execution>
+                <id>read</id>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>read-project-properties</goal>
+                </goals>
+                <configuration>
+                  <quiet>true</quiet>
+                  <files>
+                    <file>${docker.dir}/jaeger.port.properties</file>
+                    <file>${docker.dir}/infinispan.port.properties</file>
+                    <file>${docker.dir}/broker.port.properties</file>
+                    <file>${docker.dir}/jdbc-db.port.properties</file>
+                    <file>${docker.dir}/mongodb.port.properties</file>
+                    <file>${docker.dir}/qpid.port.properties</file>
+                    <file>${docker.dir}/kafka.port.properties</file>
+                    <file>${docker.dir}/auth.port.properties</file>
+                    <file>${docker.dir}/deviceregistry.port.properties</file>
+                    <file>${docker.dir}/commandrouter.port.properties</file>
+                  </files>
+                </configuration>
+              </execution>
+              <!-- 
+                set system properties adding ports as system
+                properties - so they could be resolved in .env 
+              -->
+              <execution>
+                <id>set</id>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>set-system-properties</goal>
+                </goals>
+                <configuration>
+                  <quiet>true</quiet>
+                  <properties>
+                    <property>
+                      <name>docker.host.address</name>
+                      <value>${docker.host.address}</value>
+                    </property>
+                    <property>
+                      <name>jaeger.port</name>
+                      <value>${jaeger.port}</value>
+                    </property>
+                    <property>
+                      <name>qpid.port</name>
+                      <value>${qpid.port}</value>
+                    </property>    
+                    <property>
+                      <name>kafka.port</name>
+                      <value>${kafka.port}</value>
+                    </property>    
+                  </properties>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <!-- 
+      start quarkus in dev mode on verify phase skipping tests. To read ports from docker files (if dependencies are started with tests pom) use:
+        mvn verify -Pservice-dev,read-ports
+          or
+        mvn verify -Dservice-dev
+      to use fixed ports starts with:
+        mvn verify -Pservice-dev
+          or 
+        mvn verify -Dservice-dev -P!read-ports
+    -->
+    <profile>
+      <id>service-dev</id>         
+      <activation>
```

Comment:
do we still need this?

Reply:
this is the only option I found to enable quarkus dev + read-props profile. If we decide to drop dynamic ports support this will not be needed